WordPress/src/main/java/org/wordpress/android/models/Role.java

```diff
@@ -21,21 +27,42 @@
         this.backgroundColor = backgroundColor;
     }
 
-    public static String toString(Context context, Role role) {
+    // Label to be used in UI
+    public static String getLabel(Context context, Role role) {
         return context.getString(role.label);
     }
 
-    // This method will be used to determine the role of the user from network request
-    public static Role fromString(String value) {
+    // Returns the key for that role
+    public static String toKey(Role role) {
+        switch (role) {
+            case SUPER_ADMIN:
+                return KEY_SUPER_ADMIN;
+            case ADMIN:
+                return KEY_ADMIN;
+            case EDITOR:
+                return KEY_EDITOR;
+            case AUTHOR:
+                return KEY_AUTHOR;
+            case CONTRIBUTOR:
+                return KEY_CONTRIBUTOR;
+            default:
```

Comment:
The reason I've done that way was to avoid adding an extra `return` statement after the `switch` case was done, but I understand your objection. I've made the change in [#03002cc](https://github.com/wordpress-mobile/WordPress-Android/pull/3916/commits/03002cc4e2dfb94a8d2fcf29d3048a3b70a405da) and added a comment for why we are returning "unsupported" string if no case is matched. Btw, I didn't want to return `null` because of 2 reasons.

1. It'll mean that we'll need to check for NPE when we use `Role` enum.
2. This is the whole reason for using the "unsupported".

Let me know what you think!